grammars/css.cson

```diff
@@ -498,7 +498,13 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
-        'match': '(\\.)[a-zA-Z0-9_-]+'
+        'match': '''(?x)
+          (\\.)
+          -?+                   # possessive to avoid matching '-' followed by <0-9>
+          [\\w-&&[^0-9]]        # Do NOT change 0-9 to \d.
```

Comment:
:ok_hand: Using `\d` here is actually ok most of the time, but now that the standards says 0-9 are not allowed at the beginning (but not other digits), if we want to align with the standards, we better do what it says.

Try http://rubular.com/, as oniguruma is also used in Ruby.